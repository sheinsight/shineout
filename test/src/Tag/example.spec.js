import exampleTest from '../../example'

describe('Tag[snapshot]', () => {
  exampleTest('Tag')
})
