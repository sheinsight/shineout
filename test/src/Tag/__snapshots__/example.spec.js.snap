// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tag[snapshot] Snapshot test: Tag[site/pages/components/Tag/example-1-base.tsx] 1`] = `
<div>
  <div>
    <span>
      Tag 1
    </span>
  </div>
  <div>
    <span>
      Tag 2
    </span>
  </div>
  <div>
    <div>
      <span>
        Tag 3
      </span>
    </div>
    <div>
      <svg>
        <path />
      </svg>
    </div>
  </div>
</div>
`;

exports[`Tag[snapshot] Snapshot test: Tag[site/pages/components/Tag/example-2-type.tsx] 1`] = `
<div>
  <div>
    <span>
      Default
    </span>
  </div>
  <div>
    <span>
      Success
    </span>
  </div>
  <div>
    <span>
      Info
    </span>
  </div>
  <div>
    <span>
      Warning
    </span>
  </div>
  <div>
    <span>
      Danger
    </span>
  </div>
</div>
`;

exports[`Tag[snapshot] Snapshot test: Tag[site/pages/components/Tag/example-3-bgcolor.tsx] 1`] = `
<div>
  <div>
    <span>
      #eeeeee
    </span>
  </div>
  <div>
    <span>
      #613400
    </span>
  </div>
  <div>
    <span>
      #f50
    </span>
  </div>
  <div>
    <span>
      #87d068
    </span>
  </div>
</div>
`;

exports[`Tag[snapshot] Snapshot test: Tag[site/pages/components/Tag/example-4-close.tsx] 1`] = `
<div>
  <div>
    <div>
      <span>
        onClose=true
      </span>
    </div>
    <div>
      <svg>
        <path />
      </svg>
    </div>
  </div>
  <div>
    <div>
      <span>
        onClose=function
      </span>
    </div>
    <div>
      <svg>
        <path />
      </svg>
    </div>
  </div>
  <div>
    <div>
      <span>
        onClose=promise
      </span>
    </div>
    <div>
      <svg>
        <path />
      </svg>
    </div>
  </div>
</div>
`;

exports[`Tag[snapshot] Snapshot test: Tag[site/pages/components/Tag/example-5-disabled.tsx] 1`] = `
<div>
  <div>
    <span>
      Tag 1
    </span>
  </div>
  <div>
    <span>
      Tag 2
    </span>
  </div>
  <div>
    <div>
      <span>
        Tag 3
      </span>
    </div>
    <div>
      <svg>
        <path />
      </svg>
    </div>
  </div>
</div>
`;

exports[`Tag[snapshot] Snapshot test: Tag[site/pages/components/Tag/example-6-dynamic.tsx] 1`] = `
<div>
  <div>
    <div>
      <span>
        Tag 1
      </span>
    </div>
    <div>
      <svg>
        <path />
      </svg>
    </div>
  </div>
  <div>
    <div>
      <span>
        Tag 2
      </span>
    </div>
    <div>
      <svg>
        <path />
      </svg>
    </div>
  </div>
  <div>
    <div>
      <span>
        Tag 3
      </span>
    </div>
    <div>
      <svg>
        <path />
      </svg>
    </div>
  </div>
  <div>
    <span>
      + New Tag
    </span>
  </div>
</div>
`;

exports[`Tag[snapshot] Snapshot test: Tag[site/pages/components/Tag/example-7-editable.tsx] 1`] = `
<div>
  <div>
    <div>
      <span>
        abc
      </span>
    </div>
    <div>
      <svg>
        <path />
      </svg>
    </div>
  </div>
</div>
`;
